scvi/module/_scanvae.py

```diff
@@ -323,32 +320,6 @@ def loss(
         else:
             kl_divergence_l = 0.0
 
-        if is_labelled:
```

Comment:
`feed_labels` is always `False` => `y` is always None` => `is_labelled` is always `False`